src/pages/contact.tsx

```diff
@@ -84,27 +84,28 @@ const Contact: NextPage = () => {
             <div className="gap-5 flex max-md:flex-col max-md:items-stretch max-md:gap-0">
               <div className="flex flex-col items-stretch w-[92%] max-md:w-full max-md:ml-0">
                 <div className="text-stone-900 text-2xl leading-[133.333%] w-[1068px] max-w-[1068px] max-md:max-w-full max-md:mt-10">
-                  Thank you for your interest. Please don’t hesitate to reach
-                  out to us if you need assistance or information regarding the
-                  Community Toolkit, Data Discovery Application, or The SDOH &
-                  Place Project in general. We value your input and welcome any
-                  comments you may have. Your feedback is essential in helping
-                  us improve and refine The SDOH & Place Project.
+                  Thank you for your interest. Please don&apos;t hesitate to
+                  reach out to us if you need assistance or information
+                  regarding the Community Toolkit, Data Discovery Application,
+                  or The SDOH & Place Project in general. We value your input
+                  and welcome any comments you may have. Your feedback is
+                  essential in helping us improve and refine The SDOH & Place
+                  Project.
                 </div>
               </div>
             </div>
           </div>
           <div className="flex mb-10 mt-10">
             <Link
               className="mr-10"
-              href="https://github.com/healthyregions"
-              title="@heathyregions on GitHub"
+              href="https://github.com/healthyregions/SDOHPlace"
+              title="@sdohplace on GitHub"
```

Comment:
This should still reference @HealthyRegions in the title, or something like "View code on GitHub".  There isn't an @sdohplace user on github.

Reply:
Fixed in [4f300b0](https://github.com/healthyregions/SDOHPlace/pull/50/commits/4f300b0bf06615fa1e1144a4b49d8929cb11b591)